Caboodle/Platform/Platform.android.cs

```diff
@@ -18,12 +18,15 @@ public static partial class Platform
         internal static Activity CurrentActivity =>
             lifecycleListener?.Activity;
 
-        public static void Init(Activity activity, Bundle bundle)
+        public static void Init(Application application)
```

Comment:
I feel like having multiple overloads here is confusing... I'd personally prefer the original signature with Activity and Bundle, JUST IN CASE we ever need them.  This is what Forms does as well...

Reply:
put back :) left Application overload in :)